src/redux/books/bookSlice.js

```diff
@@ -1,26 +1,7 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
```

Comment:
- [ ] Please make sure `createAsyncThunk` is imported and used in your project

Reply:
- [ ] Action creator called `fetchBooks` should exists in the `bookSlice`

Reply:
- [ ] Please use `Axios`  to make the HTTP request

Reply:
- [ ] Please make sure the `addBook` feature adds a new book via API.
- [ ] `removeBook` should remove a book via API.